### DISCUSSION TITLE
Why cordova-android@12.0.0 allows to install on Android 6.0?

### DISCUSSION THREAD

**Discussion by yoldar** [apache/cordova#426] [Q&A] [answered]
Why cordova-android@12.0.0 allows to install on Android 6.0 if minSdk must be 24? But in fact Android Studio shows minSdk 23

**Comment by breautek** [comment, marked as answer]
Is the `android-minSdkVersion` preference defined?

**Reply by yoldar** [reply]
@breautek nope =) I thought that it is by default in cordova-android@12.0.0

**Reply by breautek** [reply]
Cordova-android 12 will set Min SDK 24 by default, but there are several things that could override it.

`android-minSdkVersion` preference is one way and it's also possible that a plugin would set this. If that's the case, it would require your app to explicitly set the preference to override the plugin.

The minimum SDK can also be set via [gradle parameters](https://cordova.apache.org/docs/en/11.x/guide/platforms/android/index.html#setting-gradle-properties), which will override the cordova default.
